shanoir-ng-front/src/app/centers/center/center.component.html

```diff
@@ -57,7 +57,7 @@ <h2 class="header command-zone"i18n="Create acquisition center|Title@@centerDeta
 							{{center.street}}
 						</ng-template>
 						<ng-template ngSwitchDefault>
-							<input type="text" id="street" formControlName="street" [(ngModel)]="center.street" />
+							<input type="text" id="street" formControlName="street"/>
```

Comment:
now the formControlName has to exactly match the property name, so the auto 2 way binding works